src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -158,7 +158,7 @@ namespace AppInstaller::CLI::Workflow
             {
                 if (!agreement.Label.empty())
                 {
-                    context.Reporter.Info() << Execution::SourceInfoEmphasis << Utility::LocIndString{ agreement.Label } << " ";
+                    context.Reporter.Info() << Execution::SourceInfoEmphasis << Utility::LocIndString{ agreement.Label } << ": ";

```

Comment:
Realized that they should really have `_liv` for some magical future where I get around to enforcing localization.

Reply:
Added it to all these strings in Show. I realized there are still some non-localized strings for showing dependencies. I'll update these on my next change.